blocks/market-commentary/market-commentary.js

```diff
@@ -32,7 +37,7 @@ function createMarketCommentaryCard(cardData, placeholders) {
       { class: 'footer-row' },
       span({ class: 'footer-circle' }),
     ),
-    div({ class: 'footer-time' }, footerTimeText),
+    div({ class: 'footer-time' }, NEWS_TIME),
```

Comment:
I didn't get a chance to use the div() method. Just wondering, when the value such as `NEWS_TIME` is empty, will it still generate a div without content?

Reply:
@JiangLong2019 yes